samples/react-native/package.json

```diff
@@ -4,16 +4,28 @@
   "private": true,
   "scripts": {
     "postinstall": "patch-package",
-    "android": "react-native run-android",
-    "ios": "react-native run-ios",
```

Comment:
l: I understand that those two are not necessary on CI but I was wondering if we should keep them for local development.

Reply:
In my mind these are replaced by `yarn start` and `yarn build-*`.

There are two reasons why I don't like `react-native run-*`.

It hides some of the build logs, so ofter I have to rebuild the app anyway when the build fails.

Second, it spawns a new tab for the `dev server` which I don't like.

But it's just a personal preference.

Reply:
Thank you for the explanation @krystofwoldrich 🙇